docs/getting-started/getting-started-with-low-code-airflow.md

```diff
@@ -0,0 +1,165 @@
+---
+title: Airflow with Databricks
+id: airflow-with-databricks
+description: A tutorial on using Low-code Airflow
+sidebar_position: 7
+tags:
+  - airflow
+  - tutorial
+---
+
+Version 3.0.2 of our platform, Prophecy, introduces an exciting new feature: Low-code Airflow.
+Now, users can effortlessly create and manage Airflow Jobs using a user-friendly drag-and-drop interface.
+This empowers you to design and schedule intricate workflows without the need for coding expertise.
+The tool seamlessly translates your designs into highly optimized Python Airflow code, stored in Git, ensuring complete accessibility and openness to all users.
+Moreover, you have the flexibility to enhance functionality by incorporating your custom operators and sensors via our Gem Builder interface.
+
+#### In this quick-start, we will show you how to use Prophecy Managed Airflow to Run and schedule your Spark and SQL Pipelines
+
+We'll take you step by step from connecting your Databricks to Prophecy Managed Airflow to creating your first [Airflow DAG](https://airflow.apache.org/docs/apache-airflow/stable/core-concepts/dags.html) and scheduling it. By the end of this training, you'll have an understanding of Airflow DAGs, be able to use our Visual interface to quickly create your DAG, and schedule this DAG on Prophecy Hosted Airflow. Let's dig in!
+
+#### You will need
+
+- Databricks Account
+- A Prophecy Project With Spark Pipeline or SQL Model running on Databricks
+
+If you don't have an existing project, please check out [this guide](https://docs.prophecy.io/concepts/project/#1-create-new-project) for [setting up a Spark Project,](https://docs.prophecy.io/tutorials/videos/design-Pipeline) and [this guide](/docs/getting-started/getting-started-with-low-code-sql.md) for setting up a SQL model in Prophecy.
+
+## 1. Setup Prophecy Fabric for Airflow
+
+Prophecy introduces the concept of a Fabric to describe an execution environment. In this case, we create a Fabric to connect to Airflow, and create and schedule DAGs in it.
+For this guide, we would be using Prophecy Managed Airflow, so an external Airflow instance is not required.
+
+![Create Fabric](img/2-4-create-fabric.png)
+
+Setting up a Fabric is very straightforward. Click the **(1) Create Entity** button, and choose **(2) Create Fabric** option. The Fabric creation is composed of two steps: Basic Info and Providers setup.
+On the Basic Info screen, enter a **(1) Fabric Name**, **(2) Fabric Description**, and choose the **(3) Team** that’s going to own the Fabric.
+
+Once ready, click **(4) Continue**.
+
+![Fill_fabric_details.png](img/Airflow_1.2_Create_Fabric.png)
```

Comment:
Feedback applicable to most of the images: they're not cut properly and missing shadows! If you're really struggling with it, send Figma we can help.

Reply:
Link to figma: https://www.figma.com/file/yOcskNYqEYInn3TqzNHLrn/Prophecy-Documentation-(docs.prophecy.io)?type=design&node-id=601-1906&mode=design

I can add shadows. Just let me know what all to fix in each and I will update the images.